frontend/src/features/properties/map/activity/detail/ActivityControlsBar.tsx

```diff
@@ -17,16 +21,26 @@ export const ActivityControlsBar: React.FunctionComponent<IActivityControlsBarPr
   onEditRelatedProperties,
 }) => {
   const { hasClaim } = useKeycloakWrapper();
+  const { getOptionsByType } = useLookupCodeHelpers();
+  const activityStatusTypeOptions = getOptionsByType(API.ACTIVITY_STATUS_TYPE);
   return (
     <>
       {hasClaim(Claims.ACTIVITY_EDIT) && (
         <RightAlignedRow>
-          {!editMode && <EditButton onClick={() => setEditMode(true)} />}
+          <SectionField label="Status">
+            <Select
+              disabled={!editMode && !hasClaim(Claims.PROPERTY_EDIT)}
+              field="status"
+              options={activityStatusTypeOptions}
+            />
+          </SectionField>
```

Comment:
question: I understand that you want to disable the `<select>` when in view mode but why check for claim PROPERTY_EDIT as well? Shouldn't the activity status be driven by claim ACTIVITY_EDIT instead? Not sure if the story calls for this

Reply:
Rremoved